src/schemas/json/sarif.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2018-10-10 JSON Schema",
-  "description": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2018-10-10 JSON Schema: a standard format for the output of static analysis tools.",
+  "title": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2019-01-24 JSON Schema",
```

Comment:
>1-24 [](start = 82, length = 4)

Shouldn't this be 02-20? The two SARIF files you uploaded to src/test are labeled with 02/20.

Reply:
The answer is "Yes." This caused the AppVeyor build to fail.

---
In reply to: [259647887](https://github.com/SchemaStore/schemastore/pull/642#discussion_r259647887) [](ancestors = 259647887)

Reply:
Actually it's the other way around. Your PR title is "Schema 01-24" so it's the test files that are wrong. I see you pushed what you called the "01-24.1" schema (but whose version number is, correctly, just "01-24") to the spec repo.

---
In reply to: [259648309](https://github.com/SchemaStore/schemastore/pull/642#discussion_r259648309) [](ancestors = 259648309,259647887)

Reply:
that;s right, it's the test files that are broken.

---
In reply to: [259648588](https://github.com/SchemaStore/schemastore/pull/642#discussion_r259648588) [](ancestors = 259648588,259648309,259647887)